vignettes/basic-parliament-plots.Rmd

```diff
@@ -0,0 +1,236 @@
+---
+title: "Basic Parliament Plots"
+author: "Zoe Meers"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+source("../R/parliament_data.R")
+source("../R/helper_funcs.R")
+source("../R/draw_majorityline.R")
+source("../R/draw_legislaturelabels.R")
+library(tidyverse)
+election_data <- readRDS(file="../data/election_data.rda")
+
+```
+
+## Introduction to `ggparliament`
+
+`ggparliament` is a `ggplot2` extension for creating parliament plots. Although there are many parliament layouts around the world, we cannot create templates for all of them! Instead, we have created templated versions for the five most popular. They are:
+
+- semicircle layouts
+
+- horseshoe parliaments
+
+- opposing bench-style parliaments
+
+- the classroom layout
+
+- a circular parliament.
+
+We start with aggregate election returns by party. 
+This package comes with recent election results from the UK, Australia, the United Sates, Russia, and Germany.
+
+### Semicircle Parliament
+
+If one wanted to plot the 115th Congress in the United States, they can run the following commands:
+
+```{r}
+us_congress <- election_data %>%
```

Comment:
(I don't know what's best practice for this) is it worth explicitly loading libraries that are needed for the munging (magrittr+dplyr / tidyverse/ etc.) or letting the user load them themselves?